db/migrate/20180521181959_add_stem_offered_to_crosswalks.rb

```diff
@@ -0,0 +1,5 @@
+class AddStemOfferedToCrosswalks < ActiveRecord::Migration
+  def change
+    add_column :crosswalks, :stem_offered, :boolean
```

Comment:
Can you describe the process you're thinking through? 

My understanding is that crosswalks is an EDU provided upload, and that we're using the CSV from that file to populate this table. Are you thinking of setting stem_offered based on the process from https://github.com/department-of-veterans-affairs/gibct-data-service/issues/325#issuecomment-390193350? 

Forgive my ignorance here - I'm not 100% fluent on the details of GIDS, since I've only made relatively minor changes to it so far. Is `stem_offered` not supposed to be a new attribute on the institution?

Reply:
Yes @jkassemi here I'm planning to generate the stem_offered in crosswalks based on the logic in comment.

Reply:
So, `stem_offered` will be a new column on the `institutions` after they're built? 

If we're currently just mapping the source file to the crosswalks table, then this approach would mean we'd need to regenerate that any time either crosswalks or IPEDS was updated. Is it just possible to set the value on the institution record on rebuild so we can keep these tables representative of input data and not modifications our system is making?

Reply:
Yes @jkassemi stem_offered will be new column on the institutions will add migration as a part of this PR